core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.directive.ts

```diff
@@ -183,9 +186,10 @@ angular.module('oppia').directive('skillConceptCardEditor', [
             $scope.dragDotsImgUrl = UrlInterpolationService.getStaticImageUrl(
               '/general/drag_dots.png');
             initBindableFieldsDict();
-            $scope.$on(EVENT_SKILL_REINITIALIZED, function() {
-              initBindableFieldsDict();
-            });
+            ctrl.directiveSubscriptons.add(
```

Comment:
subscriptions is misspelled. (Make sure to proofread your PRs.)

Reply:
Changed!